src/cljs/main/broadfcui/page/billing/create_project.cljs

```diff
@@ -68,6 +68,7 @@
                            [:tr {:style {:borderTop style/standard-line}}
                             [:td {:style {:borderTop style/standard-line}}
                              [:input {:type "radio" :value (account "accountName")
+                                      :name "billingAccountSelect"
```

Comment:
When we have control over the name, I prefer `kebab-case` (hyphen-separated, not sure where I heard this term but it was recent).